etc/kayobe/environments/aufn-ceph/kolla/config/bifrost/bifrost.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Strictly this is a Jinja2 file, rather than pure YAML. Sometimes it parses ok, but if you use an if like on L8 it won't be valid YAML.

```suggestion
# yamllint disable-file
---
```